tests/foreman/cli/test_errata.py

```diff
@@ -282,60 +286,69 @@ class ErrataTestCase(CLITestCase):
     def setUpClass(cls):
         """Create 3 organizations
 
-           1- Create an org with one custom product & repository
-
-           2- Create an org2 with an other custom product & repository
-
-           3- Create an org3 with two other products, each containing one
-              custom repository
+           1. Create an org with one custom product & repository
+           2. Create an other org with an other custom product & repository
+           3. Create a multi products org with two other products, each
+              containing one custom repository
 
            note: all products repositories contain erratum
            """
         super(ErrataTestCase, cls).setUpClass()
         cls.org = make_org()
         cls.org_product = make_product(
             options={'organization-id': cls.org['id']})
-        cls.org_custom_repo = make_repository({
+        repo = make_repository({
             'download-policy': 'immediate',
             'organization-id': cls.org['id'],
             'product-id': cls.org_product['id'],
             'url': FAKE_6_YUM_REPO
         })
-        Repository.synchronize({'id': cls.org_custom_repo['id']})
-        cls.org2 = make_org()
-        cls.org2_product = make_product(
-            options={'organization-id': cls.org2['id']})
-        cls.org2_custom_repo = make_repository({
+        Repository.synchronize({'id': repo['id']})
+        cls.org_product_erratum_count = FAKE_6_YUM_ERRATUM_COUNT
+        cls.org_product_errata_id = FAKE_2_ERRATA_ID
+        # create an other org
+        cls.org_other = make_org()
```

Comment:
tbh `cls.org_other` doesn't describe its purpose. How this org is different from `cls.org` ? is it just by a product/repo/errata ? can we name it like `errata_org`